samples/src/main/java/terminal/TerminalJavaClient.java

```diff
@@ -72,6 +71,21 @@ public CompletableFuture<PaymentStatus> cancelPaymentAsync(String amountStr, Str
         return response;
     }
 
+    public RefundResponse requestRefund(String amountStr, String orderId) throws ExecutionException, InterruptedException, TimeoutException {
+        Executor delayed = CompletableFuture.delayedExecutor(5L, TimeUnit.SECONDS);
+
+        CompletableFuture<RefundResponse> completableFutureCompletableFuture = CompletableFuture.supplyAsync(() -> {
```

Comment:
Rename suggestion - will need applying to other parts of the code:
```suggestion
        CompletableFuture<RefundResponse> refundResponseFuture = CompletableFuture.supplyAsync(() -> {
```